ui/app/build-configs/directives/pnc-select-build-parameters/pncEnterBuildParameters.js

```diff
@@ -40,6 +41,14 @@
 
     // --------------------
 
+    $ctrl.updateEnumParameterValues = () => {
```

Comment:
consider such way to implement the logic: 
$ctrl.enumParameterValues = $ctrl.enumParameters.filter(parameter => parameter.name === $ctrl.key)?.[0]?.enumValues || null